### PR TITLE
[ui] tests for template editor

### PR DESCRIPTION
**Pull request by mmuzikar**
//test: `@template`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/Template.java [hunk 1]
```diff
@@ -127,9 +160,81 @@ public void uploadTemplate(String filePath) {
         flipThroughOptions();
     }
 
+    void hoverOnErrorMarker(SelenideElement marker) {
```
Comment (asmigala) [inline ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/Template.java]: Probably package-private by mistake (along with all the others)

File: ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/add/steps/TemplateSteps.java [hunk 2]
```diff
@@ -2,29 +2,73 @@
...
+    }
+
+    @Then("^there should be (\\d+) warnings in the template editor$")
+    public void checkWarningCount(int count) {
+        Assertions.assertThat(template.getAllWarnings()).hasSize(count);
+    }
+
+    @Then("^line (\\d+) should be without any warning in the template editor$")
+    public void checkLineWithoutError(int lineNum) {
+        Assertions.assertThat(template.getWarningsOnLine(lineNum)).isEqualTo(Optional.empty());
```
[the remark this answers is not in the document]
Comment (asmigala) [reply ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/add/steps/TemplateSteps.java]: Better yet, `assertThat(optional).isEmpty();`

**Comment by mmuzikar** [comment]
Thanks for the feedback! Really helpful :)

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mmuzikar** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by asmigala** [comment]
@mmuzikar  please rebase on master and run the checkstyle on your changes, it reports a bunch of wrong import order errors.

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by avano** [comment]
@mmuzikar please with the conflict so that this can be finally closed

**Comment by mmuzikar** [comment]
@avano Templates are not working properly (or at least were not when I checked it last time), after the issues with atlas map blocking this get fixed I will fix it in the migration process.

**Comment by asmigala** [comment]
What is the status of this? Can it be rebased and merged?

**Comment by mmuzikar** [comment]
@asmigala These tests are still using Angular frontend, I see no use in merging this. I'd say we can delete this and when template step starts working, I will change these editor tests as well

**Comment by asmigala** [comment]
Ok, closing

**Pull request closed without merging**